sentinel/src/transitions/accept_patient_reports.js

```diff
@@ -114,6 +114,38 @@ const addRegistrationToDoc = (doc, registrations) => {
   }
 };
 
+const addReportUUIDToRegistration = (doc, registrations) => {
+  if (registrations.length) {
+    var visitReportedDate = doc.reported_date;
+
+    var messageFound = false;
+    registrations.forEach(function(registration) {
+      if (registration.scheduled_tasks) {
+        registration.scheduled_tasks.forEach(function(task, index) {
+          if (!messageFound) {
+            if (
```

Comment:
If you change to this:

```suggestion
            if (['delivered', 'sent'].includes(task.state) && task.messages) {
```

You can have your `if` predicate on one line :-)